src/Components/CreateImageWizard/validators.ts

```diff
@@ -79,6 +79,19 @@ export const isSshKeyValid = (sshKey: string) => {
   return isPatternValid;
 };
 
+export const isPasswordValid = (password: string) => {
+  if (password === undefined) return false;
```

Comment:
`password` is a string, shouldn't the initial value be `''`?

Reply:
thank you, I think we dont need to check it at all, so I remove it, becuase user can choose to fill ssh- key or password